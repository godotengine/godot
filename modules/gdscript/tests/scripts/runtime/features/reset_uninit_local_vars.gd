# GH-89958

func test():
	if true:
		@warning_ignore("unused_variable")
		var a = 1
		@warning_ignore("unused_variable")
		var b := 1
		@warning_ignore("unused_variable")
		var c := 1

	if true:
		var a
		@warning_ignore("unassigned_variable")
		print(a)
		var b
		@warning_ignore("unassigned_variable")
		print(b)
		var c: Object
		@warning_ignore("unassigned_variable")
		print(c)
