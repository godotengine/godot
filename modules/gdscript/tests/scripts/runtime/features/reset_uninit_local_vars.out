GDTEST_OK
<null>
<null>
<null>
